node/src/node_configurator/node_configurator_standard.rs

```diff
@@ -250,10 +213,10 @@ pub fn server_initializer_collected_params<'a>(
     args: &[String],
 ) -> Result<MultiConfig<'a>, ConfiguratorError> {
     let app = app_node();
-    let user_specific_data = determine_user_specific_data(dirs_wrapper, &app, args)?;
+    let initialization_data = determine_user_specific_data(dirs_wrapper, &app, args)?;
     let config_file_vcl = match ConfigFileVcl::new(
```

Comment:
Should the constructor of `ConfigFileVcl` be modified to accept one of your new `FieldPair`s?

Reply:
I want to discuss this on the call